azure-pipelines.yml

```diff
@@ -27,13 +27,19 @@ trigger:
     include:
     - develop
     - master
+  paths:
```

Comment:
en.json updates should still build for release pushes

Reply:
We should do releases for any changes to localization IMO regardless of the language. But agree with not building for github related or api doc changes